src/lib/modules/evaluate/evaluate.go

```diff
@@ -0,0 +1,92 @@
+package evaluate
```

Comment:
Lets break this file into 3 files ith these objects:
`machine.go` => `Machine()`, `Ping`, `Jitter`, `absDiff()`
`monitor.go` => `Monitor()`, `Response`